server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -390,6 +391,10 @@ public void operation(PhysicalPlan plan) throws IOException, MetadataException {
         SetDeviceTemplatePlan setDeviceTemplatePlan = (SetDeviceTemplatePlan) plan;
         setDeviceTemplate(setDeviceTemplatePlan);
         break;
+      case AUTO_CREATE_DEVICE_MNODE:
```

Comment:
please add
```
this.templateMap.clear();
```
in the clear method to ensure complete cleanness

Reply:
Added